src/en/client_development/rust_api.md

```diff
@@ -0,0 +1,37 @@
+# Rust API
+
+If you want to develop a Rust application which interacts with Lemmy, you can directly pull in the relevant API structs. This relies on the exact code used in Lemmy, but with most heavyweight dependencies disabled (like diesel).
+
+To get started, add the following to your `Cargo.toml`:
+
+```
+[dependencies]
+lemmy_api_common = { git = "https://github.com/LemmyNet/lemmy.git" }
+```
+
+Note, at the time of writing, this code is not available on crates.io yet. You can use "0.16.3" from crates.io, but it pulls in many heavy dependencies including diesel. Best use the git dependency for now, or wait for a newer version to become available.
```

Comment:
Maybe make an issue to remind you to do that.

Reply:
I dont think an issue is necessary, because this is clear enough, and someone can fix it when they come across it. But if you want, open one.